flytekit/core/python_third_party_task.py

```diff
@@ -0,0 +1,153 @@
+from __future__ import annotations
+
+from abc import abstractmethod
+from typing import Any, Dict, Generic, List, Optional, TypeVar
+
+from flytekit.common.tasks.raw_container import _get_container_definition
+from flytekit.core.base_task import PythonTask
+from flytekit.core.context_manager import Image, ImageConfig, SerializationSettings
+from flytekit.core.resources import Resources, ResourceSpec
+from flytekit.core.tracker import TrackedInstance
+from flytekit.models import task as _task_model
+from flytekit.models.security import Secret, SecurityContext
+
+T = TypeVar("T")
+TC = TypeVar("TC")
+
+
+class TaskTemplateExecutor(TrackedInstance, Generic[T]):
+    @classmethod
+    def execute_from_template(cls, tt: _task_model.TaskTemplate, **kwargs) -> Any:
```

Comment:
i like this method, probably does not need to be a class method, this could just be a mixin?

I would drop the other 2 methods?